static/individualScreenings.js

```diff
@@ -1,10 +1,31 @@
 const currentPath = window.location.pathname;
 
-fetch("/api" + currentPath + "/screenings")
-  .then((response) => response.json())
-  .then((movieScreenings) => {
-    movieScreenings.forEach((screening) => {
-      console.log("Start time:", screening.start_time);
-      console.log("Theater:", screening.room);
-    });
+async function fetchMovieScreenings() {
+  const response = await fetch("/api" + currentPath + "/screenings");
+  const movieScreenings = await response.json();
+  return movieScreenings;
+}
+
+function renderMovieScreenings(movieScreenings) {
+  const listElement = document.querySelector(".movieInformation__list");
+  movieScreenings.forEach((screening) => {
+    const listItem = document.createElement("li");
+
+    const date = new Date(screening.start_time);
+
+    const timeOptions = { hour: "2-digit", minute: "2-digit" };
+    const dateOptions = { day: "numeric", month: "short", year: "numeric" };
+    const time = new Intl.DateTimeFormat("default", timeOptions).format(date);
+    const formattedDate = new Intl.DateTimeFormat("default", dateOptions).format(date);
+
+    listItem.innerHTML = `Theater: ${screening.room}<br>Start time: ${time}, ${formattedDate}`;
```

Comment:
I am not shure but are there alternatives to innerHTML regarding potential saftey issue with innerHTML?

Reply:
Ahh good catch i will change it to use textContent of innerHTML instead to avoid potential XSS attacks.